emukit/quadrature/acquisitions/squared_correlation.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import numpy as np
+from scipy.linalg import lapack
+from typing import Tuple
+
+from emukit.core.acquisition import Acquisition
+from emukit.quadrature.methods import VanillaBayesianQuadrature
+
+
+class SquaredCorrelation(Acquisition):
+    """
+    This acquisition function is the correlation between the integral and the new point(s) under a GP-model.
+
+    SquaredCorrelation is identical to the integral-variance-reduction acquisition up to a global normalizing constant!
+
+    .. math::
+        \rho^2(x) = \frac{(\int k_N(x_1, x)\mathrm{d}x_1)^2}{\mathfrac{v}_N v_N(x)}\in [0, 1]
+
+    where :math:`\mathfrac{v}_N` is the current integral variance given N observations X, :math:`v_N(x)` is the
+    predictive integral variance if point x was added newly, and :math:`k_N(x_1, x)` is the posterior kernel function.
+    """
+
+    def __init__(self, model: VanillaBayesianQuadrature):
+        """
+        :param model: The vanilla Bayesian quadrature model
+        """
+        self.model = model
+
+    def has_gradients(self) -> bool:
+        return True
+
+    def evaluate(self, x: np.ndarray) -> np.ndarray:
+        """
+        Evaluates the acquisition function at x.
+
+        :param x: (n_points x input_dim) locations where to evaluate
+        :return: (n_points x 1) the acquisition function value at x
+        """
+        return self.evaluate_with_gradients(x)[0]
```

Comment:
Just wandering how optimal is this. Since you don't need gradients to compute the value, perhaps we could consider a separate something liek that:

```
def evaluate:
     compute
     return value

def evaluate_with_grads:
     value = self.evaluate() 
     compute grads
```

This won't change anything for evaluate_with_grads, but may save on unnecessary computations for evaluate

Reply:
Yes, I thought about that, too. The downside is that then you have to evaluate _value_terms twice. My though was that unless you want to plot the acquisition function you probably always compute the gradients, too.